ui/components/DataTable.tsx

```diff
@@ -96,7 +96,7 @@ export const sortWithType = (rows: any[], sort: field) => {
 function UnstyledDataTable({
   className,
   fields,
-  rows,
+  rows = [],
```

Comment:
Not necessary to handle this in both places. I would say lets handle it in `sourceWithType` in case someone uses that in the future in another component.